presto-hive/src/main/java/com/facebook/presto/hive/pagefile/PageFileCorruptionException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.pagefile;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+
+public class PageFileCorruptionException
```

Comment:
Why do we need a special exception? Why not simply `PrestoException`?

Reply:
Similar as `RcFileCorruptionException` to make it clear,  do you prefer to throw `HIVE_BAD_DATA` error directly?